### PR TITLE
Updating the edit command error message

### LINKED ISSUE #190: [PE-D][Tester A] Edit tags error message does not mention no whitespace
## Problem
As a user trying to add a tag, I might include whitespace in instances likes `t/Best Friend t/Rose Seller`
However, tags actually don't accept whitespace but the error message does not make it very clear.

## Recommended Fix
Add "no whitespace" to the error message for tags.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: CaptainKeqing/ped#10

### PR DESCRIPTION
**Pull request by kailinteoo**
Resolves #190

### CODE REVIEW

File: src/main/java/seedu/address/model/tag/Tag.java [hunk 1]
```diff
@@ -9,7 +9,7 @@
  */
 public class Tag {
 
-    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
+    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric and no whitespaces is allowed";
```
Comment (rertyy) [inline src/main/java/seedu/address/model/tag/Tag.java:12]: Tag names should be alphanumeric and no whitespaces are allowed

File: docs/UserGuide.md [hunk 2]
```diff
@@ -423,7 +428,7 @@ Format: `editOrder <INDEX> [by/DEADLINE] [c/PRICE] [d/DESCRIPTION] [s/STATUS]`
   The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* There are 3 different status:
+* There are 3 different status (they are all case-insensitive):
```
Comment (rertyy) [inline docs/UserGuide.md:431]: As long as this is true

File: docs/UserGuide.md [hunk 3]
```diff
@@ -246,10 +246,13 @@ Format: `exit`
 Adds a client to BookKeeper. <br>
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
-<div markdown="span" class="alert alert-primary">
+<div markdown="block" class="alert alert-info">
```
Comment (rertyy) [inline docs/UserGuide.md:249]: Did we have a reason for changing these

Comment (kailinteoo) [reply docs/UserGuide.md:249]: The rest of the command uses <div markdown="block" class="alert alert-info">, hence I was thinking of keeping it consistent

File: docs/UserGuide.md (hunk 2 above)
Comment (kailinteoo) [reply docs/UserGuide.md:431]: yes, it accept both "PENDING" and "pending"

File: src/main/java/seedu/address/model/tag/Tag.java (hunk 1 above)
Comment (kailinteoo) [reply src/main/java/seedu/address/model/tag/Tag.java]: Edited, thanks for pointing out

**Review by rertyy — APPROVED** [review]
LGTM

**Pull request merged**